subprojects/docs/src/docs/userguide/custom_plugins.adoc

```diff
@@ -193,17 +193,17 @@ You can use the api:org.gradle.testfixtures.ProjectBuilder[] class to create api
 
 
 [[sec:using_the_java_gradle_plugin_development_plugin]]
-==== Using the Java Gradle Plugin development plugin
+==== Using the Java Gradle Plugin Development Plugin
 
-You can use the incubating <<java_gradle_plugin,Java Gradle Plugin development plugin>> to eliminate some of the boilerplate declarations in your build script and provide some basic validations of plugin metadata. This plugin will automatically apply the <<java_plugin,Java plugin>>, add the `gradleApi()` dependency to the compile configuration, and perform plugin metadata validations as part of the `jar` task execution.
+You can use the incubating <<java_gradle_plugin,Java Gradle Plugin Development Plugin>> to eliminate some of the boilerplate declarations in your build script and provide some basic validations of plugin metadata. This plugin will automatically apply the <<java_plugin,Java Plugin>>, add the `gradleApi()` dependency to the compile configuration, perform plugin metadata validations as part of the `jar` task execution, and generate plugin descriptors in the `jar` file's `META-INF` directory.
```

Comment:
"`jar` file" sounds a little off. You could say "`jar`'s output file" to be clear, but I prefer "in the resulting JAR's META-INF directory"

I also prefer to use italics for file paths unless I am referencing actual code. However, I may have to change that since I think pretty much everyone else uses code.